neuralpredictors/measures/np_functions.py

```diff
@@ -14,15 +21,137 @@ def corr(y1, y2, axis=-1, eps=1e-8, **kwargs):
         y2:      second NumPy array
         axis:    dimension(s) along which the correlation is computed. Any valid NumPy axis spec works here
         eps:     offset to the standard deviation to avoid exploding the correlation due to small division (default 1e-8)
-        **kwargs: passed to final numpy.mean operatoin over standardized y1 * y2
+        **kwargs: passed to final numpy.mean operation over standardized y1 * y2
 
     Returns: correlation array
     """
+
     y1 = (y1 - y1.mean(axis=axis, keepdims=True)) / (y1.std(axis=axis, keepdims=True, ddof=0) + eps)
     y2 = (y2 - y2.mean(axis=axis, keepdims=True)) / (y2.std(axis=axis, keepdims=True, ddof=0) + eps)
     return (y1 * y2).mean(axis=axis, **kwargs)
 
 
+def oracle_corr_corrected(repeated_outputs: ArrayLike) -> np.ndarray:
```

Comment:
We discussed that the name of the functions could be more descriptive.
If you agree, we'd call this function `conservative_oracle`

Reply:
Done!